_includes/top-menu.html

```diff
@@ -2,10 +2,11 @@
   <div class="menu-inner">
     <div class="left">
       <a class="logo" href="/"></a>
-      <a class="link-menu" href="features.html">Features</a>
-      <a class="link-menu" href="documentation.html">Documentation</a>
-      <a class="link-menu" href="pricing.html">Pricing</a>
-      <a class="link-menu" href="download.html">Download</a>
+      <a class="link-menu" href="/features.html">Features</a>
+      <a class="link-menu" href="/docs/">Documentation</a>
+      <a class="link-menu" href="/forum/">Forum</a>
+      <a class="link-menu" href="/pricing.html">Pricing</a>
+      <a class="link-menu" href="/download.html">Download</a>
```

Comment:
O my bad, so the links now are `/download/` instead of `download.html` I got confused reading `/forum/` and thought you updated just some of them!